pkg/test/utils_test.go

```diff
@@ -222,3 +199,72 @@ func find(s string, r []model.FormSubmission) bool {
 
 	return found
 }
+
+// set the environment variables for a test database
+// in the tests we are creating a unique database for testing
+// and removing it after running all the tests
+func setTestDatabase() {
+
+	// save back the MONGODB_URL env variable that it may be used for production
+	// this is sketchy and should be consider in the refactoring. We have the db package and web package getting MONGODB_URL everywhere
+	envmongodburl = os.Getenv("MONGODB_URL")
+
+	// use TEST_MONGODB_URL to create the connection to a new test database
+	mongodburl := os.Getenv("TEST_MONGODB_URL")
```

Comment:
Since we're adding a timestamp to randomize the name, I don't think we need to require another environment variable.  It seems like a small requirement to add another variable dependency, but over time the number creeps up.